lib/logqueue-fifo.c

```diff
@@ -76,13 +76,15 @@ typedef struct _InputQueue
   struct iv_list_head items;
   WorkerBatchCallback cb;
   guint16 len;
+  guint16 nfc_len;
```

Comment:
Rename `nfc` to... to something else... (also in the function names)
* `num_non_flow_controlled_msgs`
* `non_flow_controlled_len`
* ...

I think `non_flow_ctrl` also an acceptable name... 
Or, initiate a type, that could also express that we have to virtual channels...  
`log_queue_fifo_get_nfc_length(self, NON_FLOW_CONTROLLED);` (but I think a dedicated function is superior)

Reply:
Will do, thanks.

I'll also try to make this change conditional based on the config version. This seems necessary because without changing the value of `log-fifo-size()` in new configs, the memory usage can be slightly higher.